src/labels/CustomLabels.labels

```diff
@@ -2752,6 +2752,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>- Household Duplicate</value>
     </labels>
     <labels>
+        <fullName>hhErrorInvalidContactFieldName</fullName>
+        <categories>Households, Household Naming</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>hhErrorInvalidContactFieldName</shortDescription>
+        <value>The field, {0}, does not match any field name on Contact.</value>
```

Comment:
Thanks @davidhabib for the feedback. The error is displayed when Household Naming Contact fields are retrieved from ` iNaming.setHouseholdNameFieldsOnContact();` method and then is used in Contact queries throughout the system whenever Household Naming should be updated. 
The method can be implemented by a client so fields do not have to belong to any Household Naming Format field. Additionally, we would need to parse each format field or tried to execute it in order to find which one is faulty (expensive operation?).
Can we make an error more generic but still indicate it is Household Naming Settings Format field? For example:  

Invalid Household Naming Settings format: The field, FirstName}, does not match any field name on Contact.